dataverse-webapp/src/main/webapp/dataset-license-terms.xhtml

```diff
@@ -5,319 +5,55 @@
-                            </label>
-                            <div class="col-sm-9">
-                                <h:outputText value="#{MarkupChecker:sanitizeBasicHTML(termsOfUseAndAccess.contactForAccess)}" escape="false"/>
-                            </div>
-                        </div>
-                        <div class="form-group" jsf:rendered="#{!empty termsOfUseAndAccess.sizeOfCollection}">
-                            <label for="metadata_TermsUse" class="col-sm-3 control-label">
-                                #{bundle['file.dataFilesTab.terms.list.termsOfAccess.addInfo.sizeOfCollection']}
-                                <span class="glyphicon glyphicon-question-sign tooltip-icon"
-                                      data-toggle="tooltip" data-placement="auto right" data-original-title="#{bundle['file.dataFilesTab.terms.list.termsOfAccess.addInfo.sizeOfCollection.tip']}"></span>
-                            </label>
-                            <div class="col-sm-9">
-                                <h:outputText value="#{MarkupChecker:sanitizeBasicHTML(termsOfUseAndAccess.sizeOfCollection)}" escape="false"/>
-                            </div>
-                        </div>
-                        <div class="form-group" jsf:rendered="#{!empty termsOfUseAndAccess.studyCompletion}">
-                            <label for="metadata_TermsUse" class="col-sm-3 control-label">
-                                #{bundle['file.dataFilesTab.terms.list.termsOfAccess.addInfo.studyCompletion']}
-                                <span class="glyphicon glyphicon-question-sign tooltip-icon"
-                                      data-toggle="tooltip" data-placement="auto right" data-original-title="#{bundle['file.dataFilesTab.terms.list.termsOfAccess.addInfo.studyCompletion.title']}"></span>
-                            </label>
-                            <div class="col-sm-9">
-                                <h:outputText value="#{MarkupChecker:sanitizeBasicHTML(termsOfUseAndAccess.studyCompletion)}" escape="false"/>
-                            </div>
-                        </div>
+                        <ui:fragment rendered="#{!empty GuestbookTab.selectedGuestbook}">
+                            <p class="help-block">#{bundle['file.dataFilesTab.terms.list.guestbook.inUse.tip']}</p>
+                            <div class="panel panel-default">
+                                <div class="panel-body">
+                                    <p class="form-control-static pull-left">#{GuestbookTab.selectedGuestbook.name}</p>
+                                    <p:commandLink type="button" styleClass="btn btn-default pull-right"
+                                                   value="#{bundle['file.dataFilesTab.terms.list.guestbook.viewBtn']}"
+                                                   update=":datasetForm" oncomplete="PF('viewGuestbook').show();bind_bsui_components();"/>
```

Comment:
Hardcoded name of form. We shouldn't rely here that enclosing form would have this id. Probably we can use `@form` here - please check. I'm not 100% sure.

Reply:
You are right, this code is encapsulated in form id="datasetForm" so @form works.